node/baremetal/README.md

```diff
@@ -0,0 +1,418 @@
+# Bare Metal Base Node Operation
+
+The goal of this guide is to set up a fully synced Base node running on bare metal,
+including an L1 Ethereum node to support the L1 -> L2 data feed.
+
+The clients in use are as follows:
+
+- Geth - L1 Ethereum execution client
+- Prysm - L1 Ethereum consensus client
+- OP-Geth - L2 execution client for Base
+- OP-Node - L2 rollup client for Base
+
+Base is built on top of the [Optimism Bedrock stack](https://community.optimism.io/docs/developers/bedrock/)
+which is why the clients are referred to as OP-Geth and OP-Node. They are run
+with ChainID 8453, which is the L2 chain ID for Base.
+
+## Hardware Requirements
+
+For running all 4 binaries (Eth Execution layer, Eth Consensus Layer, OP-Geth,
+and OP-Node) on mainnet, we recommend the following specs:
+
+- At least 32GB of RAM
+- At least 2 TB of storage
```

Comment:
Clarify that SSDs or NVME drives are recommended. Use the language from the OP or Ethereum node pages